psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -126,6 +127,34 @@ public boolean isIgnoreBaseVoltage() {
         }
     }
 
+    private final class ShuntBlockTab {
+
+        private HashMap<Integer, Integer> ni;
+        private HashMap<Integer, Double> bi;
```

Comment:
`private final Map<Integer, Integer> ni` is better as the map are initialized in the constructor and never replaced

Reply:
OK